src/windows/FileProxy.js

```diff
@@ -168,6 +168,15 @@ WinFS.prototype.__format__ = function(fullPath) {
     return 'cdvfile://localhost' + path;
 };
 
+var windowsPaths = {
+    dataDirectory: "ms-appdata:///local/",
+    cacheDirectory: "ms-appdata:///temp/",
+    tempDirectory: "ms-appdata:///temp/",
+    syncedDataDirectory: "ms-appdata:///roaming/",
+    applicationDirectory: "ms-appdata:///",
```

Comment:
I think applicationDirectory and applicationStorageDirectory should point to ms-appx:/// rather than ms-appdata:///